helm/kube-linstor/templates/satellite-daemonset.yaml

```
@@ -131,7 +131,19 @@ spec:
           mountPath: /run/lvm
         - name: run-udev
           mountPath: /run/udev
-      {{- if .Values.configurator.enabled }}
+      {{- if .Values.satellite.monitoring.enabled }}
+      - name: drbd-reactor
+      {{- with .Values.satellite.monitoring.image }}
```

drbd-reactor depends on drbd-utils and some others, let's provide it as part of the linstor-satellite image.

```suggestion
      {{- with .Values.satellite.image }}
```